Package.swift

```diff
@@ -717,35 +701,35 @@ let package = Package(
         "FirebaseSharedSwift",
         .product(name: "GTMSessionFetcherCore", package: "GTMSessionFetcher"),
       ],
-      path: "FirebaseFunctionsSwift/Sources"
+      path: "FirebaseFunctions/Sources"
     ),
     .testTarget(
       name: "FirebaseFunctionsSwiftUnit",
```

Comment:
Nit: should this and the ones below change from `FirebaseFunctionsSwift` to just `FirebaseFunctions`, or take the old naming of `Functions`? May be easier to read in the generated Xcode schemes.

Reply:
Updated here and the schemes.